### ISSUE TITLE
Page "flashes" regularly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#52]
```
You can see this in this video aprox in second 0:32
http://www.youtube.com/watch?v=AQjW-IxUJHU
And here, that portion of the video in "slow motion"
http://www.youtube.com/watch?v=futKHlA4VBU


Well, that's it, page seems to vanish from time to time. Notice that it is the 
page itself and not a faulty screen, the title bar never disappear.
I noticed this before and without looking for it, just appears when i was 
making a video for the other issue.
This happens regularly.
I think this also happened in 1.4.0 version too.

What version of the product are you using? On what operating system?
1.2.0. Android 2.1

What device are you using?
Flytouch 2. 
A 10 inch china tablet with Android 2.1 and a Resistive one-touch screen with 
1024x600 screen resolution

Please provide any additional information below.

```

Original issue reported on code.google.com by `lanos...@gmail.com` on 9 Sep 2011 at 7:14

**Comment by GoogleCodeExporter** [comment]
```
Thanks for video.

About black screen: I see two possible causes - 
1. internal problem of the surface buffer in which we draw images.
2. sometimes all book pages are considered invisible and not painted.

We also see this effect before but it very unstable and hard to reproduce.

About slow motion: to say the truth, we never test application in such manner.

We will try to reproduce it under debug.

```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 9 Sep 2011 at 9:10
- Changed state: **Discussion**

**Comment by GoogleCodeExporter** [comment]
```
Pls try the 1.2.2-rc2
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 4 Oct 2011 at 3:47

**Comment by GoogleCodeExporter** [comment]
```
With 1.2.2-rc2 now flashes back instead white.
I noticed this:
1-After the document is opened after a few moment you start scrolling the page 
flashes black once and don't flash anymore.
2-When i double-tap to get the zoom bar, while the zoom bar is open, and you 
scroll you get one black flash and then stop flashing.
3-if you double-tap again to get rid of the zoom bar, again a few moments after 
start scrolling you get a black flash.
And so on, if you enabled or disabled zoom bar you get one black flash after 
that.
```

Original comment by `lanos...@gmail.com` on 4 Oct 2011 at 6:35

**Comment by GoogleCodeExporter** [comment]
```
Pls try the 1.2.2-rc4 version.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 5 Oct 2011 at 3:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Seems to be solved in 1.2.2-rc4 version. (At least the systematic "black 
flashes")
But another issue arise now see youtube video:
http://www.youtube.com/watch?v=Ex-qCjh2-Hk

```

Original comment by `lanos...@gmail.com` on 5 Oct 2011 at 10:27
